src/issues/components/IssueTabs.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <QTabs
+    align="justify"
+    color="secondary"
+  >
+    <QRouteTab
+      :to="{ name: 'issueChat', params: { groupId, issueId } }"
+      default
+      slot="title"
+      name="chat"
+      icon="fas fa-comments"
+    />
+    <QRouteTab
+      :to="{ name: 'issueVote', params: { groupId, issueId } }"
+      slot="title"
+      name="vote"
+      icon="fas fa-vote-yea"
+    />
+  </QTabs>
+</template>
+
+<script>
+import { mapState } from 'vuex'
+
+import {
+  QTabs,
+  QRouteTab,
+} from 'quasar'
+
+export default {
+  components: {
+    QTabs,
+    QRouteTab,
+  },
+  computed: {
+    ...mapState({
+      groupId: state => state.currentGroup.id,
```

Comment:
This is available via `currentGroup/id` getter

Reply:
It's not the same:
- `state.currentGroup.id` is set when the group _started_ loading
- `currentGroup/id` is set _after_ the group finished loading

I used this here to prevent vue-router warnings during page load. Another option would be to set `:to="null"` when the group and issue are not loaded yet.

Reply:
Perhaps the solution it to make the `currentGroup/id` getter return the same as `state.currentGroup.id` - nothing about either of those ways of getting the id would inform me of that difference currently, whereas they _seem_ like they would be the same.

In general would like to avoid depending on nuances of load ordering.